stdlib/typing_extensions.pyi

```diff
@@ -82,15 +132,14 @@ TypedDict: object
 
 OrderedDict = _Alias()
 
-if sys.version_info >= (3, 7):
```

Comment:
Technically we shouldn't do this because the change hasn't been released yet, but I don't want to have to duplicate `__all__` here.